content/correspondence/reference/API endpoints/initialize/baseCorrespondenceExt/_index.en.md

```diff
@@ -0,0 +1,99 @@
+---
+title: BaseCorrespondenceExt
+linktitle: BaseCorrespondenceExt
+description: Description of the properties
+
+weight: 60
+toc: true
+---
+
+### resourceId
+
+Type: _string_
+
+The ID for the resource associated with the correspondence
+
+### sender
+
+Type: _string_
+
+The sender of the correspondence.
+
+### sendersReference
+
+Type: _string_
+
+Used by senders and receiver to identify a specific Correspondence using external identification methods.
+
+### messageSender
+
+Type: _string_
+
+An alternative name for the sender of the correspondence. The name will be displayed instead of the organization name.
+
+### content
+
+Type: [InitializeCorrespondenceContentExt](https://github.com/Altinn/altinn-correspondence/blob/main/src/Altinn.Correspondence.API/Models/InitializeCorrespondenceContentExt.cs)
+
+The correspondence content. Contains information about the title, body, etc.
+
+### requestedPublishTime
+
+Type: _DateTimeOffset_
+
+When the correspondence should become visible to the recipient.
+
+### allowSystemDeleteAfter
+
+Type: _DateTimeOffset_
+
+The date for when Altinn can remove the correspondence from its database
+
+### dueDateTime
+
+Type: _DateTimeOffset_
+
+The date and time for when the recipient must reply.
+
+### externalReferences
+
+Type: [List\<ExternalReferencesExt>](https://github.com/Altinn/altinn-correspondence/blob/main/src/Altinn.Correspondence.API/Models/ExternalReferenceExt.cs)
+
+A list of references senders can use to tell the recipient that the correspondence is related to the referenced item(s).
+Examples include Altinn App instances, Altinn Broker file transfers.
+
+### propertyList
+
+Type: _Dictionary\<string, string>_
+
+User-defined properties related to the Correspondence.
+
+### replyOptions
+
+Type: [List\<CorrespondenceReplyOptions>](https://github.com/Altinn/altinn-correspondence/blob/main/src/Altinn.Correspondence.API/Models/CorrespondenceReplyOptionExt.cs)
+
+Options for how the recipient can reply to the correspondence.
```

Comment:
Consider adding:
"Options for how the recipient can reply to the correspondence by accessing one or more URLs."

But proper handling has yet to be implemented yet in issue: https://github.com/Altinn/altinn-correspondence/issues/527 so maybe add a comment to that issue with link to this text to ensure it is updated correctly.